app/views/users/edit.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= form_for(@user) do |f| %>
+  <div class="form-inputs">
+    <p><%= f.text_field :email %></p>
+    <p><%= f.select :role, ["admin", "member"]%></p>
```

Comment:
You should get this from `enum` methods. If in future we add another role you won't need to add it manually. (potential bug).
`User.roles.keys` - something like that. Play with it in controller.